src/Sarif.Multitool.FunctionalTests/TestData/DoNotUseFriendlyNameAsRuleId/Invalid.sarif

```diff
@@ -26,8 +26,8 @@
       "properties": {
         "expectedResults": {
           "resultLocationPointers": [
-            "runs[0].resources.rules[0]",
-            "runs[0].resources.rules[1]"
+            "runs[0].tool.rulesMetadata[0]",
```

Comment:
>rulesMetadata [](start = 26, length = 13)

I get it, I get it. Once this file is pre-release-compatibility-transformed, that's where the `resources.rules` will end up. But boy is this confusing to read. I can't wait for us to update all the test files.